source/upgrade.txt

```diff
@@ -292,45 +300,45 @@ This driver version introduces the following breaking changes:
   - ``ServerListenerAdapter``
   - ``ServerMonitorListenerAdapter``
 
-  Because of these removals, the following methods were also removed from the
+  The driver also removes the following related methods from the
   ``ConnectionPoolListener`` interface:
 
-  - ``connectionAdded``
-  - ``connectionPoolOpened``
-  - ``connectionRemoved``
+  - ``connectionAdded()``
+  - ``connectionPoolOpened()``
+  - ``connectionRemoved()``
 
-  For more information about the events package, see the `com.mongodb.event
-  package documentation <{+api+}/mongodb-driver-core/com/mongodb/event/package-summary.html>`__
+  For more information about the ``com.mongodb.event`` package, see the
+  `API documentation <{+core-api+}/com/mongodb/event/package-summary.html>`__.
 
 .. _scala-breaking-changes-v5.0-list-collections:
 
-- Adds support for a new ``authorizedCollection`` option of
-  the ``listCollections`` command. This introduces a breaking binary
-  change in the ``MongoDatabase.listCollectionNames()`` methods, meaning any
-  code using these methods must be recompiled. This change does not require any
-  changes to source code.
+- Adds the ``authorizedCollection`` option for the ``listCollections`` command.
+  This introduces a breaking binary change in the ``MongoDatabase.listCollectionNames()``
+  methods. This change does not require any changes to source code, but you must
```

Comment:
Q: Should this be "method", or is there more than one? Also applies to the next line.

Reply:
Looks like just one